web-animations/interfaces/AnimationTimeline/document-timeline.html

```diff
@@ -20,6 +21,18 @@
 }, 'document.timeline identity tests');
 
 promise_test(function(t) {
+  var baseTime = document.timeline.currentTime;
+  var iframe = document.getElementById('iframe');
+  var iframeBaseTime = iframe.contentDocument.timeline.currentTime;
+  return waitForAnimationFrames(4).then(function() {
+    var delta = document.timeline.currentTime - baseTime;
+    var iframeDelta = iframe.contentDocument.timeline.currentTime - iframeBaseTime;
+    assert_greater_than(delta, 0);
+    assert_times_equal(delta, iframeDelta, 'different document timelines should progress at the same rate');
+  });
+}, 'document.timeline should tick consistently across frames');
```

Comment:
Hi @alancutter, most of these tests look good but I wonder if this one comparing the time delta between timelines in different documents is something actually required by the spec? I don't think it is and it might be better not to require that? (e.g. so a UA could decide to throttle the ticks on an iframe which it determines if a resource-heavy advertisement etc.) What do you think?